### PR TITLE
Cleanup `win32comext.axdebug`

### PR DESCRIPTION
**Pull request by Avasam**
This is the combination of the following changes, but scoped specifically to the `win32comext.axdebug` module:
- #1994
- #2101
- #2102
- #2122 (+ manual changes)
- #2127
- #2128
- \+ comments spacing
- then ran `format_all.bat`

Merging this first will reduce changes everywhere else.
Non-obvious changes are clarified in comments.

### CODE REVIEW

File: com/win32comext/axdebug/Test/host.py [hunk 1]
```diff
@@ -97,71 +96,25 @@ def GetPathName(self):
...
-    nodes = BuildModuleTree()
-    all_real_nodes = CreateDebugDocumentHelperNodes(pdm, app, nodes)
-    AttachParentNodes(None, nodes, all_real_nodes)
-
-    parentNode = None
-    all_real_nodes = {}
-    input("Waiting...")
-    print("Done")
-
-
```
Comment (Avasam) [inline com/win32comext/axdebug/Test/host.py:157]: These were commented out in the caller code (so it's unused) and contain a bunch of undefined names / unbound variables that I can't find anywhere.
If you still think it's worth keeping around, I'll comment them out instead.

File: com/win32comext/axdebug/codecontainer.py [hunk 2]
```diff
@@ -4,17 +4,16 @@
...
 import win32api
 import winerror
-from win32com.axdebug import axdebug
+from win32com.axdebug import axdebug, contexts
+from win32com.axdebug.util import _wrap
 from win32com.server.exception import Exception
 
-from . import contexts
-from .util import RaiseNotImpl, _wrap
-
```
Comment (Avasam) [inline com/win32comext/axdebug/codecontainer.py:17]:
```
PS E:\Users\Avasam\Documents\Git\pywin32\com\win32comext\axdebug> python codecontainer.py
Traceback (most recent call last):
  File "E:\Users\Avasam\Documents\Git\pywin32\com\win32comext\axdebug\codecontainer.py", line 19, in <module>
    from . import contexts
ImportError: attempted relative import with no known parent package
```

File: com/win32comext/axdebug/codecontainer.py [hunk 3]
```diff
@@ -253,11 +252,10 @@ def GetName(self, dnt):
 
 
 if __name__ == "__main__":
-    sys.path.append(".")
-    import ttest
+    from Test import ttest
```
Comment (Avasam) [inline com/win32comext/axdebug/codecontainer.py:255]: I tested running this script from both `pywin32\com\win32comext\axdebug` and `pywin32\com\win32comext\axdebug\Test`. With this change, it finds the module in both cases.

File: com/win32comext/axdebug/documents.py [hunk 4]
```diff
@@ -34,7 +33,7 @@ def GetDocument(self):
         return self.doc
 
 
-class DebugDocumentText(gateways.DebugDocumentInfo, gateways.DebugDocumentText):
+class DebugDocumentText(gateways.DebugDocumentText):
```
Comment (Avasam) [inline com/win32comext/axdebug/documents.py:36]: MRO issue, #2071 did not do enough
```py
>>> import win32comext.axdebug.documents
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files\Python39\lib\site-packages\win32comext\axdebug\documents.py", line 37, in <module>
    class DebugDocumentText(
TypeError: Cannot create a consistent method resolution
order (MRO) for bases DebugDocumentInfo, DebugDocumentText
```

File: com/win32comext/axdebug/util.py [hunk 5]
```diff
@@ -54,24 +52,6 @@ def _wrap_debug(object, iid):
...
-def _wrap_remove(object, iid=None):
-    # Old - no longer used or necessary!
-    return
-
-
-def _dump_wrapped():
-    from win32com.server.util import unwrap
-
-    print("Wrapped items:")
-    for key, items in all_wrapped.items():
```
Comment (Avasam) [inline com/win32comext/axdebug/util.py:66]: Nothing ever adds to `all_wrapped`. Meaning this code likely does nothing.

File: Pythonwin/pywin/framework/editor/color/coloreditor.py [hunk 6]
```diff
@@ -5,13 +5,7 @@
...
 import win32con
 import win32ui
-from pywin.framework.editor import (
-    GetEditorFontOption,
-    GetEditorOption,
-    SetEditorFontOption,
-    SetEditorOption,
-    defaultCharacterFormat,
-)
+from pywin.framework.editor import GetEditorOption
```
Comment (Avasam) [inline Pythonwin/pywin/framework/editor/color/coloreditor.py:8]: Unless these were meant to be explicitly re-exported

File: Pythonwin/pywin/framework/editor/configui.py [hunk 7]
```diff
@@ -2,15 +2,7 @@
...
-from pywin.framework.editor import (
-    DeleteEditorOption,
-    GetEditorFontOption,
-    GetEditorOption,
-    SetEditorFontOption,
-    SetEditorOption,
-    defaultCharacterFormat,
-    editorTemplate,
-)
+from pywin.framework.editor import DeleteEditorOption, GetEditorOption, SetEditorOption
```
Comment (Avasam) [inline Pythonwin/pywin/framework/editor/configui.py:5]: Unless these were meant to be explicitly re-exported

File: Pythonwin/pywin/framework/editor/editor.py [hunk 8]
```diff
@@ -26,11 +26,9 @@
 from pywin.framework.editor import (
     GetEditorFontOption,
     GetEditorOption,
-    SetEditorFontOption,
-    SetEditorOption,
```
Comment (Avasam) [inline Pythonwin/pywin/framework/editor/editor.py:30]: Unless these were meant to be explicitly re-exported

**Review by mhammond — APPROVED** [review]
Thanks!

**Pull request merged**